devicetypes/smartthings/zwave-multi-metering-switch.src/zwave-multi-metering-switch.groovy

```diff
@@ -339,15 +364,24 @@ private encap(cmd, endpoint = null) {
 }
 
 private addChildSwitches(numberOfSwitches) {
+	log.debug "${device.displayName} - Executing addChildSwitches()"
 	for (def endpoint : 2..numberOfSwitches) {
 		try {
 			String childDni = "${device.deviceNetworkId}:$endpoint"
 			def componentLabel = device.displayName[0..-2] + "${endpoint}"
-			addChildDevice("Child Metering Switch", childDni, device.getHub().getId(), [
-					completedSetup	: true,
-					label			: componentLabel,
-					isComponent		: false
-			])
+			if (isWYFYTouch()) {
```

Comment:
instead of this if / else block, use something like this:
```
def childDthName = deviceIncludesMeter() ? "Child Metering Switch" : "Child Switch"
addChildDevice(childDthName, childDni, device.getHub().getId(), [
		completedSetup	: true,
		label			: componentLabel,
		isComponent		: false
])
```

Reply:
Updated